docs/README.md

```diff
@@ -0,0 +1,35 @@
+# Building Documentation
+
+## Install pre-requisites
```

Comment:
There are also sphinx docs at velox/docs/ and there exists a way to publish them to https://facebookincubator.github.io/velox/ . Would it be possible to integrate this with that ?

Reply:
Let me look into this one and see if we can get Python docs in there too.

Reply:
I updated the docs. WDYT?